charts/consul/templates/ingress-gateways-deployment.yaml

```diff
@@ -290,7 +290,10 @@ spec:
             -service-partition={{ $root.Values.global.adminPartitions.name }} \
             {{- end }}
             -log-level={{ default $root.Values.global.logLevel }} \
-            -log-json={{ $root.Values.global.logJSON }}
+            -log-json={{ $root.Values.global.logJSON }} \
+            {{- if (and $root.Values.global.metrics.enabled $root.Values.global.metrics.enableGatewayMetrics) }}
+            -telemetry-prom-scrape-path={{ $root.Values.connectInject.metrics.defaultPrometheusScrapePath }}
```

Comment:
I think we need to hard-code this one to `/metrics` (that's how it worked before). This value is controlled by the value from the `connectInject` stanza which usually does not control gateways. Also, if someone changes this value, the prometheus annotations on the gateway deployment will not change because we're not consuming this value in the gateway annotations.